modules/grpc-proto/0.0.0-20240614-43b6617/MODULE.bazel

```diff
@@ -0,0 +1,15 @@
+module(
+    name = "grpc-proto",
+    version = "0.0.0-20240614-43b6617",
+    repo_name = "io_grpc_grpc_proto",
+)
+
+bazel_dep(name = "rules_proto", version = "4.0.0")
+bazel_dep(name = "protobuf", version = "23.1", repo_name = "com_google_protobuf")
+# -- bazel_dep definitions -- #
+
+http_archives_extension = use_extension("//bazel:http_archives_extension.bzl", "http_archives_extension")
+use_repo(http_archives_extension, "com_google_googleapis")
```

Comment:
can we use the googleapis from bzlmod instead? i saw on one repo that not doing that lead to ODR issues since you get the bzlmod one from grpc itself

Reply:
yea seems like we should